_shared/hooks/post_gen_project.py

```diff
@@ -144,12 +169,13 @@ def main():
         template_ignore_patterns.extend([
             "{{ cookiecutter.package_name }}/__init__.py",
             "{{ cookiecutter.package_name }}/app.py",
+            "{{ cookiecutter.package_name }}/models/__init__.py",
             "Dockerfile",
             "package.json",
             "yarn.lock",
             ".docker.env",
-            "{{ cookiecutter.package_name }}/pshell.py",
```

Comment:
I've _removed_ `pshell.py` from `template_ignore_patterns`: I think we actually do want `make template` to update this file, I think it contains standard contents that we probably want to be exactly the same across projects. (Projects can always add `pshell.py` to their own per-project ignore lists in `cookiecutter.json` if they want to do their own thing.)